runners/llama-server.go

```diff
@@ -1,4 +1,4 @@
-package llm
+package runners
```

Comment:
Similar to above, I'd probably keep this PR focused on renaming gpu to `discover`